schema/encoding/jsonschema/benchmark_test.go

```diff
@@ -42,11 +42,13 @@ func BenchmarkEncoder(b *testing.B) {
 			Schema: studentSchema,
 		},
 	}
+	buf := new(bytes.Buffer)
 	for i := range testCases {
 		tc := testCases[i]
 		b.Run(tc.Name, func(b *testing.B) {
 			for i := 0; i < b.N; i++ {
-				enc := jsonschema.NewEncoder(new(bytes.Buffer))
+				buf.Truncate(0)
```

Comment:
Same here, we still need to allocate a new buffer to avoid benchmarks affecting each other in a bad way.